reference/filesystem/functions/basename.xml

```diff
@@ -31,6 +31,8 @@
     <function>basename</function> bierze pod uwagę obecnią lokalizację systemu, więc aby widzieć
     prawidłowe basename ze ścieżek zawierających znaki spoza alfabetu łacińskiego, musi zostać
     ustawiona odpowiednia lokalizacja przy pomocy funkcji <function>setlocale</function>.
+    Jeśli parametr <parameter>path</parameter> zawiera znaki, które są nieprawidłowe dla
+    bieżących ustawień regionalnych, zachowanie <function>basename</function> jest niezdefiniowane.
```

Comment:
To jest coś co na 100% chcę opisać w readme (glossary, czy jakkolwiek to nazwiemy), wydaje mi się że trzeba tutaj coś dobrze pomyśleć. Zakładam że tłumaczyłeś frazę "locale settings". Nie wiem czy istnieje jakieś powszechnie używane tłumaczenie w j. polskim, więc ja tłumaczyłem na "ustawienia locale", co brzmi fatalnie ale przynajmniej pozwala wyszukać coś w Google na ten temat.

Co Ty na to, żeby połączyć Twoje lepsze tlumaczenie z moim i tłumaczyć to jako "ustawień regionalnych (locale)"? Wtedy mógłbym to z czystym sumieniem dać do tego glossary bo chyba łączy zalety obu.

Reply:
Super pasuje mi to "ustawień regionalnych (locale)" i oczywiście będzie to w README / Glossary.